### ISSUE TITLE
[Bug]: RadioGroup widget accepts duplicate values

### ISSUE THREAD

**Issue by aswathkk** [appsmithorg/appsmith#9875]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The following options is not throwing any error 
```
[
  {
    "label": "Yes",
    "value": "YES"
  },
  {
    "label": "No",
    "value": "NO"
  },
  {
    "label": "No",
    "value": "NO"
  }
]
```

### Steps To Reproduce

Enter duplicate values into the options.

### Environment

Production

### Version

Cloud

**Comment by wmdev0808** [comment]
@somangshu ,
This issue has already been addressed by PR #10012

**Comment by aswathkk** [comment]
Thanks, @wmdev0808 
This issue got resolved in release
